R/tm_t_mult_events.R

```diff
@@ -280,14 +280,23 @@ template_mult_events <- function(dataname,
 #' @export
 #'
 #' @examples
-#' adsl <- tmc_ex_adsl
-#' adcm <- tmc_ex_adcm
+#' ADSL <- tmc_ex_adsl
+#' ADCM <- tmc_ex_adcm
 #' adcm_keys <- c("STUDYID", "USUBJID", "ASTDTM", "CMSEQ", "ATC1", "ATC2", "ATC3", "ATC4")
 #'
 #' app <- teal::init(
 #'   data = cdisc_data(
-#'     cdisc_dataset("ADSL", adsl),
-#'     cdisc_dataset("ADCM", adcm, keys = adcm_keys)
+#'     ADSL = ADSL,
+#'     ADCM = ADCM,
+#'     code = "
+#'       ADSL <- tmc_ex_adsl
+#'       ADCM <- tmc_ex_adcm
+#'     ",
+#'     join_keys = join_keys(
+#'       join_key("ADSL", keys = get_cdisc_keys("ADSL")),
+#'       join_key("ADCM", keys = adcm_keys),
+#'       join_key("ADCM", "ADSL", keys = c("USUBJID", "STUDYID"))
+#'     )
```

Comment:
Please create `join_keys` object before teal::init and include in teal init. Please avoid `get_cdisc_keys` I don't think it will survive long.
```r
join_keys <- default_cdisc_join_keys[c("ADSL", "ADCM")
join_keys[["ADCM"]][["ADCM"]] <- adcm_keys
```

```suggestion
#'     join_keys = join_keys,
```